src/test/resources/data/stubbed/callCredit.csv

```diff
@@ -0,0 +1,2 @@
+title,firstname,surname,flatNumber,streetNumber,street,town,postcode,sortCode,accountNumber
+MRS,ANNIE,MCLAREN,,1,FIA LANE,TEST TOWN,X9 9AB,999998,29250380
```

Comment:
can these be changed to random variables?

Reply:
I think there s a convention of doing this in csv files that the perf test runner can use as random numbers

Reply:
Not written the stub for this yet , but I think we could randomise it